pkg/util/xds/stats_callbacks.go

```diff
@@ -31,10 +38,13 @@ type statsCallbacks struct {
 	NoopCallbacks
 	responsesSentMetric    *prometheus.CounterVec
 	requestsReceivedMetric *prometheus.CounterVec
+	versions               *prometheus.GaugeVec
```

Comment:
nit: probably makes sense to name it `versionsMetric` to be aligned with other fields of this struct